frontend/homeTabs.jsx

```diff
@@ -37,38 +42,50 @@ export function DiagnosticsTab() {
   const heartRateDiagnostics = sampleData.map((user) => user.heartRate);
 
   return (
-    <View style={styles.container}>
-      <Text>Heart Rate Analysis</Text>
-      <LineChart
-        data={{ labels: daysOfTheWeek, datasets: [{ data: heartRateDiagnostics }] }}
-        height={200}
-        width={Dimensions.get('window').width - 60}
-        fromZero
-        withShadow={false}
-        chartConfig={{
-          backgroundGradientFrom: '#A6CDF0',
-          backgroundGradientTo: '#CFDFF7',
-          color: (opacity = 1) => `rgba(0,0,0, ${opacity})`,
-        }}
-      />
-    </View>
+    <PageView>
+      <PageTitle>Diagnostics</PageTitle>
+      <CardView>
+        <LinearGradient colors={['#F9F6FF', '#CFDFF7']} style={{ padding: 20, paddingTop: 0, borderRadius: 10 }}>
+          <CardTitle>Heart Rate Analysis</CardTitle>
+          <LineChart
+            data={{ labels: daysOfTheWeek, datasets: [{ data: heartRateDiagnostics }] }}
+            height={200}
+            width={Dimensions.get('window').width - 60}
+            fromZero
+            withShadow={false}
+            chartConfig={{
+              backgroundGradientFrom: '#A6CDF0',
+              backgroundGradientTo: '#CFDFF7',
+              color: (opacity = 1) => `rgba(0,0,0, ${opacity})`,
```

Comment:
This line contains an unnecessary argument `opacity`. @samika1 is there a particular reason you implemented the chart in this way?
```suggestion
              color: () => 'rgba(0,0,0,1)',
```

Reply:
@vinaynagappala You might also wanna consider committing this to remove the unnecessary argument.